header.go

```diff
@@ -57,29 +58,18 @@ func NewHeader(cards []Card, htype HDUType, bitpix int, axes []int) *Header {
 	}
 
 	if _, ok := keys["NAXIS"]; !ok {
-		dcards = append(dcards, Card{
+		dcards = append(cards, Card{
```

Comment:
```suggestion
		dcards = append(dcards, Card{
```
otherwise duplicates would occur.